From d933f07a6db754a7846add1de763c30ffa61c171 Mon Sep 17 00:00:00 2001
From: ZeNiXxX <vitjahermann@gmail.com>
Date: Tue, 21 Aug 2018 13:53:13 +0200
Subject: [PATCH 12/17] Speed Up Animation & UI

Speed things up to the max

Change-Id: Id3ae0a2a9fcba58de031fc6e855b72356973fbcc

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

android.view: Speed things up!!

Speed up Orientation Listener

Signed-off-by: Chet Kener <Cl3Kener@gmail.com>
Signed-off-by: Eliminater74 <eliminater74@gmail.com>
-----------------------------------------------------

modified and squashed by @ZeNiXxX
speed up animation
Signed-off-by: KillerDroid96 <mkeller96@gmail.com>
---
 core/res/res/values/config.xml          |  1 +
 packages/SystemUI/res/values/config.xml | 18 +++++++++---------
 2 files changed, 10 insertions(+), 9 deletions(-)

diff --git a/core/res/res/values/config.xml b/core/res/res/values/config.xml
index c57b5c0567a..a6553479235 100644
--- a/core/res/res/values/config.xml
+++ b/core/res/res/values/config.xml
@@ -3278,6 +3278,7 @@
          animations. Note that this is only a default and will be overridden by a
          user-set value if toggled by settings so the "Transition animation scale" setting
          should also be hidden if intended to be permanent. -->
+
     <item name="config_appTransitionAnimationDurationScaleDefault" format="float" type="dimen">1.0</item>
 
     <!-- Flag indicates that whether non-system apps can be installed on internal storage. -->
diff --git a/packages/SystemUI/res/values/config.xml b/packages/SystemUI/res/values/config.xml
index e860f098395..62bd8695197 100644
--- a/packages/SystemUI/res/values/config.xml
+++ b/packages/SystemUI/res/values/config.xml
@@ -51,7 +51,7 @@
     <!-- Vibrator pattern for camera gesture launch. -->
     <integer-array translatable="false" name="config_cameraLaunchGestureVibePattern">
         <item>0</item>
-        <item>400</item>
+        <item>80</item>
     </integer-array>
 
     <!-- How many icons may be shown at once in the system bar. Includes any
@@ -138,8 +138,8 @@
     <bool name="quick_settings_rssi_tile_capitalization">true</bool>
 
     <!-- Timeouts for brightness dialog to disappear -->
-    <integer name="quick_settings_brightness_dialog_short_timeout">2000</integer>
-    <integer name="quick_settings_brightness_dialog_long_timeout">4000</integer>
+    <integer name="quick_settings_brightness_dialog_short_timeout">1000</integer>
+    <integer name="quick_settings_brightness_dialog_long_timeout">3000</integer>
 
     <!-- Should "4G" be shown instead of "LTE" when the network is NETWORK_TYPE_LTE? -->
     <bool name="config_show4GForLTE">true</bool>
@@ -158,7 +158,7 @@
     <integer name="heads_up_notification_minimum_time">2000</integer>
 
     <!-- milliseconds before the heads up notification accepts touches. -->
-    <integer name="touch_acceptance_delay">700</integer>
+    <integer name="touch_acceptance_delay">400</integer>
 
     <!-- The duration in seconds to wait before the dismiss buttons are shown. -->
     <integer name="recents_task_bar_dismiss_delay_seconds">400</integer>
@@ -167,16 +167,16 @@
     <integer name="recents_task_enter_from_app_duration">200</integer>
 
     <!-- The duration for animating the task decorations in after transitioning from an app. -->
-    <integer name="recents_task_enter_from_affiliated_app_duration">125</integer>
+    <integer name="recents_task_enter_from_affiliated_app_duration">75</integer>
 
     <!-- The duration for animating the task decorations out before transitioning to an app. -->
-    <integer name="recents_task_exit_to_app_duration">125</integer>
+    <integer name="recents_task_exit_to_app_duration">75</integer>
 
     <!-- The min animation duration for animating the nav bar scrim in. -->
-    <integer name="recents_nav_bar_scrim_enter_duration">400</integer>
+    <integer name="recents_nav_bar_scrim_enter_duration">100</integer>
 
     <!-- The animation duration for scrolling the stack to a particular item. -->
-    <integer name="recents_animate_task_stack_scroll_duration">200</integer>
+    <integer name="recents_animate_task_stack_scroll_duration">100</integer>
 
     <!-- The delay to enforce between each alt-tab key press. -->
     <integer name="recents_alt_tab_key_delay">200</integer>
@@ -189,7 +189,7 @@
     <bool name="recents_grow_in_multiwindow">true</bool>
 
     <!-- Animation duration when using long press on recents to dock -->
-    <integer name="long_press_dock_anim_duration">250</integer>
+    <integer name="long_press_dock_anim_duration">180</integer>
 
     <!-- Recents: The relative range of visible tasks from the current scroll position
          while the stack is focused. -->
-- 
2.19.1

